tools/run_tests/run_tests.py

```diff
@@ -57,8 +57,8 @@ def __init__(self, config, environ={}):
     self.allow_hashing = (config != 'gcov')
     self.environ = environ
 
-  def job_spec(self, binary, hash_targets):
-    return jobset.JobSpec(cmdline=[binary],
+  def job_spec(self, cmdline, hash_targets):
```

Comment:
How about writing a doc string for this method, complete with "Args" and "Returns"? I'm worried that basic things like its input and return types are only tracked in your head.